brdgrd.c

```diff
@@ -125,10 +128,14 @@ int rewrite_win_size( unsigned char *packet ) {
 	 */
 	VRB("Window size before rewriting: %u\n", ntohs(tcphdr->window));
 
-	/* randomize window size within [60,90] to prevent fingerprinting */
-	do {
-		new_window = 60 + (rand() % 31);
-	} while (new_window < 60 || new_window > 90);
+	/* randomize window size within [min,max] to prevent fingerprinting */
+	if (window_rnd_mask) {
+		do {
+			new_window = window_min + (rand() % window_rnd_mask);
+		} while (new_window < window_min || new_window > window_max);
+	} else {
+		new_window = window_min;
```

Comment:
Isn't this going to be 0 if no windows are given over the command line? I think we should either have a valid default window, or require the window command line arguments.

Reply:
Default window_min is 60 and window_max is 90, they're not in BSS, they are initialized.
So `min: 60 (0x3c), max: 90 (0x5a), delta 30 (0x1e), mask 31 (0x1f)`.

Reply:
You're right, my bad!